addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -1526,7 +1526,7 @@ X[]
                             await deleteBackward(editor);
                             await insertText(editor, 'x');
                         },
-                        contentAfterEdit: '<div><p>cd</p><br><span>x[]</span></div>',
+                        contentAfterEdit: '<div><p>cd</p><br><span data-oe-zws-empty-inline="">x[]\u200B</span></div>',
```

Comment:
Well that's kind of the thing with not cleaning zws in the sanitizer : we don't clean them until we save. I tried and it worked alright though because we have code to ignore zws. But I'm also not sure it's the best idea to not clean zws in the sanitizer. What I'm sure of though is we can't remove the code from `cleanForSave` so if we keep the sanitizer code then we have that duplication and that's it. Personally, I can live with it ^^'

Reply:
I think removing it is worth the try. Let's trust our support of zws in the wild and we'll see how it goes. If it becomes a problem then we can start by fixing our zws support and as a last resort we will still be able to put back some zws cleaning into sanitize.